pontoon/teams/templates/teams/teams.html

```diff
@@ -58,10 +66,16 @@ <h1>
 <section id="main" class="clearfix">
   <div class="container">
     <menu class="controls">
-      <div class="search-wrapper clearfix">
+      <div class="search-wrapper small clearfix">
```

Comment:
We should still show the full size search bar if the user is not authenticated, to be consistent with the `/projects/` page. Or show the small one there, too.